### PR TITLE
Vale integration for markdown formatting and styling

### PR DESCRIPTION
**Pull request by fcabrera23**
- Initial Vale configuration - For more information, check https://github.com/errata-ai/vale-action
- Microsoft Vale styling - For more information, check https://github.com/errata-ai/Microsoft
- GitHub Actions CI/CD configuration to add PR review
- Added Vale instructions to README.MD
- Updated support statement in SUPPORT.MD
- Starting with warning level for recommendations - Consider moving it to suggestions in the future.

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -1,25 +1,13 @@
...
 
 This project uses GitHub Issues to track bugs and feature requests. Please search the existing
-issues before filing new issues to avoid duplicates.  For new issues, file your bug or
+issues before filing new issues to avoid duplicates. For new issues, file your bug or
 feature request as a new Issue.
 
-For help and questions about using this project, please **REPO MAINTAINER: INSERT INSTRUCTIONS HERE
-FOR HOW TO ENGAGE REPO OWNERS OR COMMUNITY FOR HELP. COULD BE A STACK OVERFLOW TAG OR OTHER
-CHANNEL. WHERE WILL YOU HELP PEOPLE?**.
+If you need help or have questions about using this project, please visit [Arc Jumpstart Mission](https://arcjumpstart.com/mission) and [Arc Jumpstart FAQ](https://arcjumpstart.com/FAQ). If you still have doubts or concerns, you can file your question as an Issue.
```
Comment (likamrat) [inline SUPPORT.md:9]:
```suggestion
If you need help or have questions about using this project, please visit [Arc Jumpstart Mission](https://arcjumpstart.com/mission) and [Arc Jumpstart FAQ](https://aka.ms/Jumpstart-FAQ). If you still have doubts or concerns, you can file your question as an Issue.
```

File: .github/styles/Microsoft/AMPM.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+extends: existence
+message: Use 'AM' or 'PM' (preceded by a space).
+link: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/date-time-terms
```
Comment (likamrat) [inline .github/styles/Microsoft/AMPM.yml:3]:
```suggestion
link: https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/date-time-terms
```

1. remove all "en-us" everywhere
2. Update all docs links to learn

**Comment by fcabrera23** [comment]
@likamrat added suggested changes - Also, pipeline ran correctly and no markdown errors need to be addressed as part of this PR

**Review by likamrat — APPROVED** [review]
LGTM

**Review by likamrat — APPROVED** [review]
LGTM

**Pull request merged**